app/views/collections/ugc.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module Collections
+  class Ugc < ApplicationView
+    include BrowsableView
+    include SearchableView
+
+    def js_vars
+      [
+        {
+          name: 'pageName', value: 'e7a_1418'
+        }
+      ]
+    end
+
+    def page_title
```

Comment:
Could you instead implement `#page_content_heading` returning just the title of the firstworldwar collection if it exists, otherwise falling back to hard-coded  'First World War', leaving the addition of the site title to `ApplicationView#page_title`.